publisher/internal/analytics/ethereum/operations.go

```diff
@@ -128,9 +151,13 @@ func (rem *Removal) Process(burn WrappedEventLog) error {
 	}
 	rem.Position = *remPosition
 
-	rem.OperationBase.includeTokenPrices(&rem.Position) // 6) Getting token prices
-	rem.Position.calculatePosition()                    // 7) Save Liquidity Entry and Liquidity Pool
+	rem.OperationBase.includeTokenPrices(&rem.Position)
```

Comment:
It's a little bit confusig what `includeTokenPrices` does and why it requires a pointer to `Position`.
Maybe it could be improved if  `includeTokenPrices` wouldn't modify a position, but do some calculations and return a new position that then could be assigned to `rem.Position`. 

Or maybe it could even clearer if split into two parts - one retrieves relevant token prices(could be `OperationBase` method), and another updates the Position(could be `Position` method)?